### ISSUE TITLE
Add control methods for new trace_list features

### ISSUE THREAD

**Issue by mghenderson** [anl-cyberscience/FlexTransform#48]
Add ability to enable/disable `trace_list` features throughout FlexT.  Currently it only operates on the `DEBUG` level, but several commands output even if there are no elements in the list
Example:
```
FlexTransform.Config (/Users/mhend/git/FlexTransform/FlexTransform/Configuration/Config.py:41) DEBUG:Initializing Config with trace_list of 0 elements
```

**Comment by mghenderson** [comment]
@ChrisStrasburg Am I reading this right?  The trace index will create a dictionary of things to trace, with each value in the top-level dictionary containing the dictionary it was originally created in?

For example, given a single source field _time_ via the CLI to trace, the `self.trace_list` object will look like this:
```python
self.trace_list = {
    "time": {
        "src_fields": ["time"],
        "dst_fields": [],
        "src_IRIs": [],
        "dst_IRIs": []
    }
}
```
Is this complex structure something you use?  Most of the references I've seen where simple checks to see if a value existed as a key for `self.trace_list`.

**Comment by mghenderson** [comment]
Fixed in PR #54
